GW2EIEvtcParser/EncounterLogic/Fractals/SunquaPeak/AiKeeperOfThePeak.cs

```diff
@@ -408,5 +410,45 @@ internal override void CheckSuccess(CombatData combatData, AgentData agentData,
                     throw new MissingKeyActorsException("Ai not found");
             }
         }
+
+        protected override void SetInstanceBuffs(ParsedEvtcLog log)
+        {
+            base.SetInstanceBuffs(log);
+            IReadOnlyList<AbstractBuffEvent> dwd = log.CombatData.GetBuffData(AchievementEligibilityDancingWithDemons);
```

Comment:
Just to be sure, both requires to be done on a full Ai fight?

Reply:
Yes, energy dispersal can't be achieved in CM unless it's both phases, otherwise has to be done in normal mode